src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier{
+    Random random = new Random();
```

Comment:
Consider making 'random' a private final field, as it is a common practice to encapsulate such fields.

Reply:
Consider making 'random' a private final field, as it is a common practice to encapsulate such fields.